src/AccordionContainer/AccordionContainer.tsx

```diff
@@ -129,7 +128,7 @@ export class Provider extends React.Component<ProviderProps, ProviderState> {
             () => {
                 if (this.props.onChange) {
                     this.props.onChange(
-                        this.props.accordion
```

Comment:
It probably makes more sense to flip the ternary here than to negate the condition, for readability's sake. Doesn't really matter though, as I've[ proposed some changes](https://github.com/springload/react-accessible-accordion/issues/156) to this so it's likely going to change in the near future anyway.

Reply:
Have just changed this (flipped the ternary rather than negating the condition) - it should be one of the changes in the `chore/unify-onChange-behaviour` branch, which I've just submitted a pull request on.